src/AutoRest.CSharp/Common/Output/Models/LongRunningOperation.cs

```diff
@@ -17,6 +16,13 @@ namespace AutoRest.CSharp.Output.Models.Requests
 {
     internal class LongRunningOperation : TypeProvider
     {
+        protected LongRunningOperation(BuildContext context) : base(context)
+        {
+            Description = string.Empty;
+            DefaultName = string.Empty;
+            DefaultAccessibility = string.Empty;
+        }
+
         public LongRunningOperation(OperationGroup operationGroup, Input.Operation operation, BuildContext context, LongRunningOperationInfo lroInfo) : base(context)
         {
             Debug.Assert(operation.IsLongRunning);
```

Comment:
The original constructor was written based on the fact that input operation **is** LRO. However with `NonLRO` now inherits it, I found it very difficult to reuse, so I came up with an additional _protected_ constructor.

Reply:
This to me suggests that this modeling is wrong, and we should rethink it.